src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Common.Test/BackgroundDocumentGeneratorTest.cs

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT license. See License.txt in the project root for license information.
 
+#nullable disable
```

Comment:
Would it be possible to leave test projects as defaulting to nullable disable instead of on a per-file basis?

Reply:
It's possible, but the only reason I can think of to not do that is simplicity in cases where there is no automated refactoring.

Reply:
o interesting, does the refactoring not take into account the project settings?

Reply:
No, I just mean the refactoring is the reason why I was able to easily update the test projects. Fully enabling NRT would be the goal, and the only difference between production and test code is prioritization of the work.